client/components/profile-friends.jsx

```diff
@@ -51,11 +56,17 @@ export default class ProfileFriends extends React.Component {
 
   acceptRequest(sendId, sendUsername, index) {
     const tokenJSON = localStorage.getItem('token');
-    fetch(`api/friendAccept/${sendId}`, {
+    const newInfo = {
+      sendId,
+      sendUsername
+    };
+    fetch('api/friendAccept', {
```

Comment:
`fetch` request paths should always start with a `/`. So here should be `'/api/friendAccept'`